.eslintrc.json

```diff
@@ -1,16 +1,21 @@
 {
     "env": {
         "browser": true,
-        "webextensions": true
+        "webextensions": true,
+        "jest/globals": true
     },
     "extends": [
+        "plugin:jest/recommended",
         "plugin:import/recommended",
         "plugin:import/react",
         "plugin:promise/recommended",
         "plugin:react/recommended",
         "standard",
         "standard-react"
     ],
+    "globals": {
+        "db": true
+    },
```

Comment:
Ok, not sure how the mock system works; I would find it weird if it adds such globals implicitly. Anyway, you could add a line `/* global db */` to the file to suppress the error. Or it should be in a config only used for tests, because it is important to have this warning enabled in the rest of the code.